synapse/tools/autodoc.py

```diff
@@ -258,6 +258,27 @@ def main(argv, outp=None):
                 if pdoc:
                     rst.addLines('\t\t- %s' % (pdoc,))
 
+        # Add universal props last
+        if None in props:
```

Comment:
while it's def not a perf issue, this has the same "if foo in bar:  bar.get(foo)" convention which causes double hash calculations and lookups which i try to avoid all together rather than splitting hairs over where to use them and where not to :D

Reply:
if autodoc ever becomes a perf bottleneck i'll but a nice scotch for everyone using synapse ever :cocktail: